src/Microsoft.ML.Data/Transforms/ExtensionsCatalog.cs

```diff
@@ -84,11 +84,11 @@ public static ColumnSelectingEstimator DropColumns(this TransformsCatalog catalo
             => ColumnSelectingEstimator.DropColumns(CatalogUtils.GetEnvironment(catalog), columnsToDrop);
 
         /// <summary>
-        /// ColumnSelectingEstimator is used to select a list of columns that user wants to keep from a given input.
+        /// <see cref="ColumnSelectingEstimator"/> is used to select a list of columns that user wants to keep from a given input.
         /// </summary>
         /// <remarks>
         /// <format type="text/markdown">
-        /// <see cref="SelectColumns"/> operates on the schema of an input IDataView,
+        /// <see cref="ColumnSelectingEstimator"/> operates on the schema of an input IDataView,
```

Comment:
><see cref="ColumnSelectingEstimator"/> [](start = 12, length = 38)

not your doing, but for markdown we have to go xref:

<xref:Microsoft.ML.Transforms.ColumnSelectingEstimator>